projects/igniteui-angular/src/lib/grids/column.component.ts

```diff
@@ -1025,6 +1042,54 @@ export class IgxColumnComponent implements AfterContentInit {
             }
         }
     }
+
+    /**
+     * @hidden
+     */
+    getGridTemplate(isRow, isIE) {
+        const itemAccum = isRow ?
+            (acc, val) => Math.max(val.rowStart + val.gridRowSpan - 1, acc) :
+            (acc, val) => Math.max(val.colStart + val.gridColumnSpan - 1, acc);
+        const templateItems = this.children && this.children.reduce(itemAccum, 1) || 1;
+        const generatedSizes = !isRow ? this.generateColumnSizes(this.children) : null;
+        return isIE ?
+        generatedSizes || `(1fr)[${templateItems}]` :
+            generatedSizes || `repeat(${templateItems},1fr)`;
+    }
+
+    protected generateColumnSizes(children) {
```

Comment:
add return type

on that note, the word generate isn't usually associated with methods that return something